lib/travis/build/addons/srcclr.rb

```diff
@@ -0,0 +1,29 @@
+require 'shellwords'
+
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Srcclr < Base
+        SUPER_USER_SAFE = true
+
+        def before_finish
+          srcclr_config = @config.to_s.strip.shellescape
```

Comment:
Use `config` method to access the addon's configuration.

What is the rationale for `#to_s`? If the user passes `true` or `false` from the YAML file, that value should be passed to the addon as is. It is possible that the user passes something like `TrUe`, but that is a string with those characters, so the user is *expecting* that to be valued as such.